kustomize/base/operators/spark/kustomization.yaml

```diff
@@ -1,6 +1,7 @@
 resources:
 - spark-operator.yaml
-- spark-dummy-crd.yaml
+- sparkapplications-crd.yaml
```

Comment:
we should probably add this to a CRD section

Reply:
resolving as we decided to have this as a resource